backend/api/src/handlers/dynamodb/put-service-client.ts

```diff
@@ -34,8 +34,8 @@ export type putServiceHandlerInvokeEvent = {
     body: string;
 };
 
-export const putServiceClientHandler = async (event: putServiceHandlerInvokeEvent): Promise<APIGatewayProxyResult> => {
-    const payload: clientRegistryRegistrationResponse = JSON.parse(event.body);
+export const putServiceClientHandler = async (event: APIGatewayProxyEvent): Promise<APIGatewayProxyResult> => {
+    const payload = JSON.parse(event.body as string);
```

Comment:
Same as above but I previously did some type annotation to give a bit more type safety to what the body of the event looked like, not sure why it's been removed